src/data/faq.json

```diff
@@ -467,20 +467,26 @@
                                     "<b>Limitation</b>",
                                     "With the new function, only PDF documents of certificates that were also issued in Germany can be generated. Otherwise, a detailed message appears: 'Display print version not possible - The print version of the certificate cannot be displayed because the certificate was not issued in Germany.'",
-                                    "See also the blog article: <a href='../../blog/2021-09-22-cwa-version-2-10/' target='_blank' >Blog article Corona-Warn-App version 2.10</a>",
+                                    "Unlike the export of individual certificates, the export of all certificates is not limited to those issued in Germany, taking into account all EU digital COVID certificates.",
                                     "<b>Application</b>",
                                     "After successfully exporting the certificate to the PDF document, you can decide whether you want to save it on your smartphone, import it to other apps or print it. To do this, please use the standard functions of the operating system.",
-                                    "<b>Step-by-step instructions:</b>",
-                                    "To create a printout of your EU COVID Digital Certificate, please follow these steps: <ul><li>Tap on the 'Certificates' tab in the Corona-Warn-App</li><li>Select a person, by tapping on a corresponding certificate in the certificate overview</li><li>Scroll down and select a certificate of this person</li><li>In this detail view, tap on the three-dot menu in the upper right corner (Android) or on 'More' at the bottom (iOS)</li><li>Tap on 'Display print version'</li><li>Tap on 'Continue'</li><li>Tap on the icon for 'Print' or for 'Share' in the upper right corner</li><li>In the following view, you can print the created PDF document using the standard functions of the operating system, import it into other apps or even share it</li></ul>"
+                                    "<b>Step-by-step instructions: Exporting a single certificate</b>",
+                                    "To create a printout of your individual EU COVID Digital Certificate, please follow these steps: <ul><li>Tap on the 'Certificates' tab in the Corona-Warn-App</li><li>Select a person, by tapping on a corresponding certificate in the certificate overview</li><li>Scroll down and select a certificate of this person</li><li>In this detail view, tap on the three-dot menu in the upper right corner (Android) or on 'More' at the bottom (iOS)</li><li>Tap on 'Display print version'</li><li>Tap on 'Continue'</li><li>Tap on the icon for 'Print' or for 'Share' in the upper right corner</li><li>In the following view, you can print the created PDF document using the standard functions of the operating system, import it into other apps or even share it</li></ul>",

```

Comment:
With 2.24.x and Android 12 I am seeing "Next" not "Continue"

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2961/commits/7e085cdd9b7b63f28941b34fb0eacc970ebf8d85, thanks for your suggestion